### PR TITLE
CNFT1-3395: Add patient feature flag

### PR DESCRIPTION
**Pull request by joeaxcdw**
## Description

Add a new feature flag `patient.add.enabled` to control the url and route for `/patient/add`. Also make backend changes to configuration API.

## Tickets

* [Jira Ticket](https://cdc-nbs.atlassian.net/browse/CNFT1-3395)

## Checklist before requesting a review
- [X] PR focuses on a single story
- [X] Code has been fully tested to meet acceptance criteria
- [X] PR is reasonably small and reviewable (Generally less than 10 files and 500 changed lines)
- [X] All new functions/classes/components reasonably small
- [X] Functions/classes/components focused on one responsibility
- [X] Code easy to understand and modify (clarity over concise/clever)
- [X] PRs containing TypeScript follow the [Do's and Don'ts](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html)
- [X] PR does not contain hardcoded values (Uses constants)
- [X] All code is covered by unit or feature tests

### CODE REVIEW

File: apps/modernization-ui/src/configuration/defaults.ts [hunk 1]
```diff
@@ -33,6 +33,7 @@ const defaultFeatures: Features = {
     search,
     patient: {
         add: {
+            enabled: false,
```
Comment (benlam-ignw) [inline apps/modernization-ui/src/configuration/defaults.ts:36]: use the type Toggle = { enabled: boolean } here

File: apps/modernization-ui/src/configuration/configuration.ts [hunk 2]
```diff
@@ -46,6 +46,7 @@ type Features = {
     search: Search;
     patient: {
         add: {
+            enabled: boolean;
```
Comment (benlam-ignw) [inline apps/modernization-ui/src/configuration/configuration.ts:49]: use the type Toggle = { enabled: boolean } here

**Review by benlam-ignw — APPROVED** [review]

File: apps/modernization-ui/src/configuration/configuration.ts (hunk 2 above)
Comment (benlam-ignw) [reply apps/modernization-ui/src/configuration/configuration.ts:49]: it looks like there is a mix of both being used here besides this story so not necessary to change

**Review by mpeels — APPROVED** [review]

**Review by adamloup-enquizit — COMMENTED** [review]
Could you add 

```cucumber
| patient add     | enabled  |
| patient add     | disabled |
```

to [the options-api cucumber tests?](https://github.com/CDCgov/NEDSS-Modernization/blob/main/libs/configuration-api/src/test/resources/features/Features.feature).

This will verify that the API pulls the values from configuration.  This is an instance where coverage percentage doesn't indicate assertions

**Review by hclarkEnq — DISMISSED** [review]

**Review by adamloup-enquizit — DISMISSED** [review]
👍

**Review by stevegsa — APPROVED** [review]

**Review by alaapbharadwaj — APPROVED** [review]

**Review by mpeels — APPROVED** [review]

**Pull request merged**